fluentlenium-core/src/main/java/org/fluentlenium/core/domain/FluentList.java

```diff
@@ -259,6 +260,16 @@ public String getText() {
         return null;
     }
 
+    @Override
+    public FluentList<FluentWebElement> find(String name) {
+        return find(name, new Filter[0]);
+    }
+
+    @Override
+    public FluentList<FluentWebElement> find(String name, Predicate<FluentWebElement>... predicates) {
+        return null;
```

Comment:
It doesn't make sense to me. Could you please elaborate? This is one of the reason why we won't to merge this pull request.